Source/DataSources/exportKml.js

```diff
@@ -323,54 +340,34 @@ function exportKml(options) {
 }
 
 function createKmz(kmlString, externalFiles) {
```

Comment:
If these changes look good, I'll make similar ones in `KmlDataSource` and this should be good to merged once the worker config is figured out.

Reply:
Cool, I had a few comments. Let me know when `KmlDataSource` is ready and I'll take a look.